repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -3,19 +3,29 @@ package cromwell.pipeline.datastorage.dto
 import java.nio.file.Path
 
 import play.api.libs.functional.syntax._
-import play.api.libs.json.{ Format, Json, OFormat }
+import play.api.libs.json._
 import slick.lifted.MappedTo
 
 final case class Project(
   projectId: ProjectId,
   ownerId: UserId,
   name: String,
   repository: String,
+  visibility: Visibility = Private,
```

Comment:
fields with default values should be placed at the end of the list

Reply:
Oh, my mistake - fixed.